docs/PARSING_DATA.md

```diff
@@ -0,0 +1,216 @@
+# Parsing Data
+
+Telegraf has the ability to take data in a variety of formats. Telegraf requires
+configuration from the user in order to correctly parse, store, and send the
+original data. Telegraf does not take the raw data and maintain it internally.
+
+Telegraf uses [line protocol][] to store data. This means that data needs to be
```

Comment:
Hmmm that's not completely true. Maybe
```suggestion
Telegraf uses an internal metric representation consisting of the metric name, tags, fields and a timestamp. This means that data needs to be
```

Reply:
I like your suggestion and I will accept it! However, at the same time, aren't those essentially the definition of line protocol? I like to mention line protocol since that tends to be a helpful way to show or have them related to how the data looks.

Reply:
I agree to mention the line-protocol, maybe something like
```text
Telegraf uses an internal metric representation consisting of the metric name, tags, fields and a timestamp inspired by the [line protocol][].
```